infrastructure/src/main/java/org/corfudb/infrastructure/DataStore.java

```diff
@@ -14,14 +14,17 @@
 import java.nio.file.Paths;
 import java.nio.file.StandardCopyOption;
 import java.nio.file.StandardOpenOption;
-import java.util.Arrays;
-import java.util.Map;
+import java.util.*;
```

Comment:
Conventionally we avoid wildcard imports.

Reply:
Done.